src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -65,6 +52,10 @@ def kubeconfig_path(self):
     def iso_download_path(self):
         return self._config.iso_download_path
 
+    @property
+    def was_existed(self) -> bool:
```

Comment:
maybe add a docstring for what ``was_existed`` mean

Reply:
I renamed it `is_installed`, we set the flag to `True` after the cluster being actually installed or when the cluster is  already installed by an existing infraenv.

Reply:
yes it's a better name
if it could be computed that would be even better, but that's not a must